tests/integration/pybaseball/test_standings.py

```diff
@@ -19,7 +19,7 @@ def get_division_counts_by_season(season: Optional[int]) -> int:
 
 class TestBRefStandings:
     @pytest.mark.parametrize(
-        "season", [(x) for x in range(1871, most_recent_season() + 1)] + [(None)] # type: ignore
+        "season", [(x) for x in range(1871, most_recent_season() - 1)]
```

Comment:
The `-1` here should be incorrect, no? Through `most_recent_season()` in the range = MRS - 1?

Reply:
Yep you're right, will fix.